samples/nrf9160/aws_fota/README.rst

```diff
@@ -31,56 +112,133 @@ Building and running
+      nrf_inbuilt_key_write => result=0
+      nrf_inbuilt_key_write => result=0
+      nrf_inbuilt_key_write => result=0
+      LTE Link Connecting ...
+      LTE Link Connected!
+      IPv4 Address 0xeeb73a0d
+      client_id: nrf-aws-fota
+      [mqtt_evt_handler:129] MQTT client connected!
+
+   (?any data in there that should not be public or that is confusing?)
+#. Log on to the `AWS IoT console`_, go to **Manage** > **Things**, and select your thing.
+#. Go to **Shadow** and confirm that the application version (``nrfcloud__dfu_v1__app_v``) is the one that you configured for the sample.
+#. In the :ref:`configuring`, change the application version.
+   Then rebuild the application, but do not program it.
+#. Go to `firmware.nrfcloud.com`_ and sign in.
+   If you already have an nRF Cloud account, you can use the same credentials.
+   Otherwise, create an account and sign in.
+#. In the menu on the left-hand side, select **Firmware**.
+#. Click **Generate new** to create a URL for the firmware image.
+#. Click **Upload** for the URL that you created and select the file ``app_update.bin`` (located in the ``zephyr`` subfolder of your build directory).
+#. Create a job document (a text file) with the following content, replacing *host_url* with the server part of the URL that you created (for example, ``s3.amazonaws.com``) and *file_path* with the path and file name (for example, ``/nordic-firmware-files/0943dfbf-cb10-4eb7-8277-a8b179eaf4ff``):
+   (?should we have an actual file here, or just state ``/nordic-firmware-files/XXXXXXX``?)
+
+   .. parsed-literal::
+      :class: highlight
+
+      {
+        "operation": "app_fw_update",
+        "fwversion": "v1.0.2",
+        "size": 181124,
+        "location": {
+          "protocol": "http:",
+          "host": "*host_url*",
+          "path": "*file_path*"
+         }
+      }
+
+   See `AWS IoT Developer Guide: Jobs`_ for more information about AWS jobs.
+#. Log on to the `AWS S3 console`_.
+#. If you do not have a bucket yet, create one with the default settings.
+#. Select the bucket, click **Upload**, and select your job document.
+   Use the default settings when uploading the file.
+#. Log on to the `AWS IoT console`_, go to **Manage** > **Jobs**, and select **Create a job**.
+#. Click **Create custom job** and enter a unique job ID.
+   Select your device and the job file that you uploaded to AWS S3.
+   Use the default settings for all other options.
+#. Since the sample is configured to subscribe to the ``app_fw_update`` job topic (?what is it actually subscribing?), it picks up the job automatically.
+   This can take several minutes.
+   Select the job in AWS IoT to confirm that it is in progress.
+   Note that it might show as in progress on the overview, while it is actually queued.
+#. In the terminal emulator, observe that the new firmware image is downloaded and installed.
+   You should see output similar to the following::
+
+      (?is there any output worth mentioning?)
```

Comment:
Not sure they'll get some progress output at least which is some in the line of  `downloading fragment` and `writing x bytes to flash` I can find the exact output if it's needed.

Also it could be good to see that it subscribed to `notify-next` when it connected to AWS as an output. It would state something in the line of `subscribed to: $aws/things/<thing-name>/jobs/notify-next`